pkg/models/model_studio.go

```diff
@@ -56,3 +101,12 @@ func (s *Studios) Append(o interface{}) {
 func (s *Studios) New() interface{} {
 	return &Studio{}
 }
+
+// TODO: Seems like a good candidate for generics
+type StudioDBInput struct {
+	StudioCreate *Studio
+	StudioUpdate *StudioPartial
+
+	ParentCreate *Studio
+	ParentUpdate *StudioPartial
+}
```

Comment:
This is definitely a different architecture than currently exists, but for calls to Create I needed a StudioCreate, and maybe a ParentCreate or ParentUpdate. For calls to UpdatePartial, I needed a StudioUpdate and maybe a ParentCreate or ParentUpdate.
I tried a few different things like passing them in as params, or stuffing the parent objects into the studio objects, and passing flags for what to create/update... but this seemed to work the best.

Reply:
See my comment in the graphql changes. The operation needs to be split up. Even if we disregard my comments there, I don't think this is a good solution to what you're trying to do. The `sqlite` code shouldn't be responsible for updating related objects; that's the job of the calling code.